### DISCUSSION TITLE
Templating: Mixed data source for adding variables

### DISCUSSION THREAD

**Discussion by Prasoon6667** [grafana/grafana#38376] [Dashboards, Panels & Visualizations]
For querying, grafana have mixed datasource option. It would be good to allow this option in variable section too.

Use case: I have apps running on different clusters and I have prometheus for each cluster as metric server. I need to create a generic dashboard that will show all the datas from all the clusters.  I have deployed various versions of app in a single cluster for different customers so for querying I am using variables to select app names and the query will show the respective app data . With this I can achieve single cluster app datas to be in a single dashboard. Defining variable doesn't supports multiple datasources and I can't show it in a single dashboard.

**Comment by josephcomisi** [comment]
Any news? this feature would be really helping a lot.
This has been missing for a while now: https://github.com/grafana/grafana/issues/10224 :)

Any way to bypass it by creating two variables and merge the results in another variable?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
